### PR TITLE
Oopsie Woopsie

### PR DESCRIPTION
**Pull request by steets250**
Set the wrong version for `react-router` and busted the site. Site is now unbusted.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "react-infinite-scroller": "^1.2.4",
     "react-lazy-load-image-component": "^1.5.0",
     "react-redux": "^7.1.0",
-    "react-router": "^4.3.1",
+    "react-router": "^5.0.0",
```
Comment (StormFireFox1) [inline package.json:31]: Is this line really necessary? I don't recall us having it before.

**Review by StormFireFox1 — COMMENTED** [review]
I'm confused a bit, but otherwise I'll take your word on the un-busting.

File: package.json (hunk 1 above)
Comment (steets250) [reply package.json:31]: It was a missing dependency. If I didn't include, it would pop up every time we ran yarn.

Comment (StormFireFox1) [reply package.json:31]: Hmm. Likely `npm` popped it in without being explicitly requested?

Comment (steets250) [reply package.json:31]: Screen Shot 2020-10-27 at 8 33 39 PM
`npm` doesn't warn on install, but those packages have been missing.

**Review by leestanley — APPROVED** [review]

**Pull request merged**